plugins/inputs/uwsgi/README.md

```diff
@@ -0,0 +1,90 @@
+# uWSGI
+
+The uWSGI input plugin gathers metrics about uWSGI using its [Stats Server](https://uwsgi-docs.readthedocs.io/en/latest/StatsServer.html).
+
+### Configuration
+
+```toml
+[[inputs.uwsgi]]
+  ## List with urls of uWSGI Stats servers. Url must match pattern:
+  ## scheme://address[:port]
+  ##
+  ## For example:
+  ## servers = ["tcp://localhost:5050", "http://localhost:1717", "unix:///tmp/statsock"]
+  servers = ["tcp://127.0.0.1:1717"]
+
+  ## General connection timout in seconds
+  # timeout = 5
+```
+
+
+### Metrics:
+
+ - uwsgi_overview
+  - tags:
+    - url
```

Comment:
Let's change this to the `source` tag.  For unix sockets use os.Hostname().  For now, don't worry about port/protocol/path.  Let's add this to all metrics, not just the ones that already have it.